test/src/test/java/org/corfudb/integration/CorfuReplicationSiteConfigIT.java

```diff
@@ -342,11 +343,11 @@ public void runSiteSwitch() throws Exception {
             sleep(sleepInterval);
 
             siteManager.prepareSiteRoleChange();
-            while (replicationStatus != CorfuReplicationManager.PERCENTAGE_BASE) {
-                replicationStatus = siteManager.queryReplicationStatus();
-                System.out.print("\nreplication percentage done " + replicationStatus);
-                sleep(sleepInterval);
-            }
+//            while (replicationStatus != CorfuReplicationManager.PERCENTAGE_BASE) {
+//                replicationStatus = siteManager.queryReplicationStatus();
```

Comment:
Is this required but still not working? or old code we can remove?

Reply:
It's working. Brought back.